Some example cases are failed

Example cases for a single struct are failed.

```sh
% go test -v .
=== RUN   ExampleNew
--- PASS: ExampleNew (0.00s)
=== RUN   ExampleBasic
--- PASS: ExampleBasic (0.00s)
=== RUN   ExampleSetAlignment_Field
--- PASS: ExampleSetAlignment_Field (0.00s)
=== RUN   ExampleSetOption_Field
--- PASS: ExampleSetOption_Field (0.00s)
=== RUN   ExampleToTable_struct
--- FAIL: ExampleToTable_struct (0.00s)
got:
+-------------------------+
|          Person         |
|-------------------------|
|  key     |  value       |
|----------+--------------|
|  age     |  31          |
|  height  |  1.8         |
|  name    |  Ruby Cohen  |
|  male    |  true        |
+----------+--------------+
want:
+------------------------------------------+
|                  Person                  |
|------------------------------------------|
|  age  |  height  |     name     |  male  |
|-------+----------+--------------+--------|
|  31   |  1.8     |  Ruby Cohen  |  true  |
+-------+----------+--------------+--------+
=== RUN   ExampleToTable_anonymousStruct
--- FAIL: ExampleToTable_anonymousStruct (0.00s)
got:
field is nil.
+-----------------------------------------+
|           docker log file info          |
|-----------------------------------------|
|  key   |  value                         |
|--------+--------------------------------|
|  path  |  /var/log/system.d/docker.log  |
|  size  |  1.8                           |
|  mod   |  511                           |
+--------+--------------------------------+
want:
+-------------------------------------------------+
|               docker log file info              |
|-------------------------------------------------|
|              path              |  size  |  mod  |
|--------------------------------+--------+-------|
|  /var/log/system.d/docker.log  |  1.8   |  777  |
+--------------------------------+--------+-------+
=== RUN   ExamplePrint
--- PASS: ExamplePrint (0.00s)
=== RUN   ExamplePrintWithCaption
--- PASS: ExamplePrintWithCaption (0.00s)
=== RUN   ExampleToTable_structArray
--- PASS: ExampleToTable_structArray (0.00s)
=== RUN   ExampleToTable_anonymousStructArray
--- PASS: ExampleToTable_anonymousStructArray (0.00s)
=== RUN   ExampleToTable_map
--- PASS: ExampleToTable_map (0.00s)
=== RUN   ExampleToTable_mapArray
--- PASS: ExampleToTable_mapArray (0.00s)
=== RUN   ExampleToTable_json
--- PASS: ExampleToTable_json (0.00s)
=== RUN   ExampleToTable_jsonArray
--- PASS: ExampleToTable_jsonArray (0.00s)
=== RUN   ExampleCSVToTable
--- PASS: ExampleCSVToTable (0.00s)
FAIL
FAIL    github.com/ecoshub/stable       0.234s
FAIL
```

using Go 1.18

```sh
% go version
go version go1.18 darwin/amd64
```

<details>
<summary>go env</summary>
<pre>
<code>
% go env
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/Users/matsuyoshi/Library/Caches/go-build"
GOENV="/Users/matsuyoshi/Library/Application Support/go/env"
GOEXE=""
GOEXPERIMENT=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/matsuyoshi/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/matsuyoshi/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/local/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/darwin_amd64"
GOVCS=""
GOVERSION="go1.18"
GCCGO="gccgo"
GOAMD64="v1"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/matsuyoshi/projects/ghq/github.com/ecoshub/stable/go.mod"
GOWORK=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -arch x86_64 -m64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/3c/mm_kj9h959lgv1gm6_cht0340000gp/T/go-build592897154=/tmp/go-build -gno-record-gcc-switches -fno-common"
</code>
</pre>
</details>

--- a/encoding_structs.go
+++ b/encoding_structs.go
@@ -47,22 +47,12 @@ func structToTable(s interface{}) (*STable, error) {
 	caption := t.Name()
 	table := New(caption)
 
-	table.AddFieldWithOptions("key", &Options{
-		HeaderAlignment: AlignmentLeft,
-	})
-
-	table.AddFieldWithOptions("value", &Options{
-		HeaderAlignment: AlignmentLeft,
-	})
-
 	fieldNames := getFieldNames(v, t)
-	fieldValues := getFieldValues(v, t)
+	table.AddFields(fieldNames...)
 
-	for i := range fieldNames {
-		key := fieldNames[i]
-		value := fieldValues[i]
-		table.Row(key, value)
-	}
+	fieldValues := getFieldValues(v, t)
+	table.Row(fieldValues...)
+
 	return table, nil
 }
 

